covid19.datacommons.io/etlMapping.yaml

```diff
@@ -0,0 +1,34 @@
+mappings:
+  - name: covid19
+    doc_type: location
+    type: aggregator
+    root: location
+    props:
+      - name: submitter_id
+      - name: project_id
+      - name: country_region
+      - name: province_state
+      - name: latitude
+      - name: longitude
+    flatten_props:
+      - path: time_series
+        props:
+          - name: confirmed
+          - name: deaths
+          - name: recovered
+          - name: testing
+          - name: date
```

Comment:
It looks like only the latest timeseries record is ETLed for each location. We can either
- keep it that way, but definitely put everything in the same tab (Location), because it's not a timeseries without the time
- update the etl mapping to ETL all the dates